Win32.xs

```diff
@@ -1132,6 +1132,40 @@ XS(w32_GetOSVersion)
     PUTBACK;
 }
 
+XS(w32_GetOSUpdateBuildRevision)
+{
+    dXSARGS;
+    LONG status;
+    DWORD val, val_size = sizeof(val);
+    PFNRegGetValueA pfnRegGetValueA;
+    HMODULE module;
+    
+    if (items)
+        Perl_croak(aTHX_ "usage: Win32::GetOSUpdateBuildRevision()");
+
+    EXTEND(SP, 1);
+
+    module = GetModuleHandleA("advapi32.dll");
+    GETPROC(RegGetValueA);
+    if (!pfnRegGetValueA)
+        XSRETURN_UNDEF;
+
+    status = pfnRegGetValueA(
+        HKEY_LOCAL_MACHINE,
+        "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion",
+        "UBR",
+        RRF_RT_REG_DWORD,
+        NULL,
+        &val,
+        &val_size
+    );
```

Comment:
This makes no sense or is a copy paste from SO/reddit. `RegGetValueA` and  `advapi32.dll` are WinNT 3.0 Gold 1993 functions.  `GetModuleHandleA` and `RegGetValueA` are wrong unless you need ABI compatibility with Win95 (not NT 4.0), use the `W` variants instead.